app/facades/foods_facade.rb

```diff
@@ -0,0 +1,19 @@
+class FoodsFacade
+  def self.search_food(query)
+    foods = []
+    data = FoodsService.search_food(query)
+    data[:foods].first(10).each do |food|
```

Comment:
This is exactly where I would like to see the JSON scoped down to the `first(10)` objects. This way we only ever create the objects the view needs, instead of thousands of objects.